ci/build_deploy_infrastructure.yaml

```diff
@@ -0,0 +1,20 @@
+platform: linux
```

Comment:
- Elsewhere we use `-` rather than `_` in filenames. Also, I can't see why we've called this "Build and deploy infrastructure" in the original pipeline, rather than just "Deploy infrastructure".
- In other tasks the image required for the task is configured in the task, is there a reason that's not the case for this task?

Reply:
To the first point - I'll change to `-` and simplify the name :+1: 

Second point - this task requires an image that has an env var expansion in the path. This didn't work in the task file, so I have specified it in the pipeline instead. From the docs it looks like this could be done in the task file on a newer concourse version using vars.

Reply:
@ajmaddaford can I resolve this?

Reply:
Can we use the image in the task now that we're using v5 of Concourse and can resolve vars in external tasks?

Reply:
Yes - done :+1: